app/src/main/java/com/tr4n/moviedb/ui/home/TabMovieViewModel.kt

```diff
@@ -1,6 +1,49 @@
 package com.tr4n.moviedb.ui.home
 
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.tr4n.moviedb.data.model.Movie
+import com.tr4n.moviedb.data.source.MoviesRepository
+import kotlinx.coroutines.launch
 import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
 
-class TabMovieViewModel : ViewModel(), KoinComponent
+class TabMovieViewModel : ViewModel(), KoinComponent {
+
+    private val moviesRepository : MoviesRepository by inject()
+
+    var listMoviesNowPlaying = MutableLiveData<List<Movie>>()
+    var exception = MutableLiveData<Exception>()
+
+    fun getTabMovie(tab : String, language : String, page : Int) {
+        viewModelScope.launch {
+            try {
+                listMoviesNowPlaying.value = moviesRepository.getTabMovie(tab, language, page).results ?: emptyList()
```

Comment:
tách dòng ra, không viết dòng quá dài 
```suggestion
                val nowPlayingMovies = moviesRepository.getTabMovie(tab, language, page).results ?: emptyList()
                listMoviesNowPlaying.value = nowPlayingMovies
```